packages/terra-alert/tests/wdio/alert-spec.js

```diff
@@ -65,7 +65,7 @@ Terra.describeViewports('Alert', ['tiny', 'large'], () => {
     it('alert content is focused when rendered with an action element', () => {
       browser.url('/raw/tests/cerner-terra-core-docs/alert/custom-prop-alert');
 
-      browser.keys(['Tab', 'Tab', 'Tab', 'Tab', 'Enter']);
+      browser.keys(['Tab', 'Tab', 'Tab', 'Tab', 'Tab', 'Tab', 'Enter']);
```

Comment:
why are the additional tab key is required to bring focus to alert content..?

Reply:
@supreethmr Earlier pressing tab key was not focusing inside the input field, that is the deselect option. Now since there are two options in the list, to get out side of the input box we need to press `tab` two more times